### ISSUE TITLE
feat: Development Docs

### ISSUE THREAD

**Issue by bkeane** [python-poetry/poetry-plugin-export#230]
Hello!

I would like to develop a new output type. Can provide as a PR when done.

In embarking on this, I'm scratching my head a bit on development flow. Any chance someone could append a `development` section to the `README.md`? I will figure it out regardless, but the assist out-of-the-gate would be lovely for myself and future devs who want to put their weight behind this!

If I figure out a tidy enough flow I will PR that as well. But if someone out there has a "this is how I implement in this repo" already in mind 🙏 .

Thanks!

**Comment by radoering** [comment]
Are you looking for something like [this](https://python-poetry.org/docs/contributing#local-development)?

**Comment by bkeane** [comment]
Yes! No need for anywhere near as much polish. Primary answers I've slowly started looking for all surround:

"How can I clone this repo, write some code in it, and 1) run tests, 2) run it within poetry itself for integration ease?"

**Comment by robertlagrant** [comment]
This is a general gap for plugin development in Poetry at the moment, I think.
